cli/args/flags.rs

```diff
@@ -759,76 +759,80 @@ fn clap_root() -> Command {
-    .subcommand(coverage_subcommand())
-    .subcommand(doc_subcommand())
-    .subcommand(eval_subcommand())
-    .subcommand(fmt_subcommand())
-    .subcommand(init_subcommand())
-    .subcommand(info_subcommand())
-    .subcommand(install_subcommand())
-    .subcommand(uninstall_subcommand())
-    .subcommand(lsp_subcommand())
-    .subcommand(lint_subcommand())
-    .subcommand(repl_subcommand())
     .subcommand(run_subcommand())
-    .subcommand(task_subcommand())
-    .subcommand(test_subcommand())
-    .subcommand(types_subcommand())
-    .subcommand(upgrade_subcommand())
-    .subcommand(vendor_subcommand())
+    .defer(|cmd| {
+      cmd
+        .subcommand(bench_subcommand())
+        .subcommand(bundle_subcommand())
+        .subcommand(cache_subcommand())
+        .subcommand(check_subcommand())
+        .subcommand(compile_subcommand())
+        .subcommand(completions_subcommand())
+        .subcommand(coverage_subcommand())
+        .subcommand(doc_subcommand())
+        .subcommand(eval_subcommand())
+        .subcommand(fmt_subcommand())
+        .subcommand(init_subcommand())
+        .subcommand(info_subcommand())
+        .subcommand(install_subcommand())
+        .subcommand(uninstall_subcommand())
+        .subcommand(lsp_subcommand())
+        .subcommand(lint_subcommand())
+        .subcommand(repl_subcommand())
+        .subcommand(task_subcommand())
+        .subcommand(test_subcommand())
+        .subcommand(types_subcommand())
+        .subcommand(upgrade_subcommand())
+        .subcommand(vendor_subcommand())
+    })
     .long_about(DENO_HELP)
     .after_help(ENV_VARIABLES_HELP)
 }
 
 fn bench_subcommand() -> Command {
-  runtime_args(Command::new("bench"), true, false)
-    .arg(check_arg(true))
-    .arg(
-      Arg::new("json")
-        .long("json")
-        .action(ArgAction::SetTrue)
-        .help("UNSTABLE: Output benchmark result in JSON format"),
-    )
-    .arg(
-      Arg::new("ignore")
-        .long("ignore")
-        .num_args(1..)
-        .use_value_delimiter(true)
-        .require_equals(true)
-        .help("Ignore files")
-        .value_parser(value_parser!(PathBuf)),
-    )
-    .arg(
-      Arg::new("filter")
-        .long("filter")
-        .allow_hyphen_values(true)
-        .help("Run benchmarks with this string or pattern in the bench name"),
-    )
-    .arg(
-      Arg::new("files")
-        .help("List of file names to run")
-        .num_args(..)
-        .value_parser(value_parser!(PathBuf))
-        .action(ArgAction::Append),
-    )
-    .arg(
-      Arg::new("no-run")
-        .long("no-run")
-        .help("Cache bench modules, but don't run benchmarks")
-        .action(ArgAction::SetTrue),
-    )
-    .arg(watch_arg(false))
-    .arg(no_clear_screen_arg())
-    .arg(script_arg().last(true))
-    .about("Run benchmarks")
-    .long_about(
-      "Run benchmarks using Deno's built-in bench tool.
+  Command::new("bench").defer(|cmd| {
```

Comment:
Might be wrong, but it seems that `.defer` causes the command description to disappear ( see #19554)